dev.sh

```diff
@@ -43,7 +43,7 @@ echo
 echo -e "\033[32m:::::::::: PLEASE WAIT for @opencrvs/client ::::::::::\033[0m"
 echo
 sleep 10
-export LANGUAGES=$1
+export LANGUAGES="en"
```

Comment:
workflow microservice requires a comma delimited string of languages in order to validate if the user has submitted names in all languages.  As the default config has only English, I think we can hardcode this in the dev script.  But there is technical debt regarding languages that we need to tackle at some point.

Reply:
https://github.com/opencrvs/opencrvs-core/issues/2477